scripts/bench_cfg.sh

```diff
@@ -6,6 +6,7 @@ DEFAULT_BM_HEAP_PAGES=4096
 PROJECT_ROOT=${PROJECT_ROOT:-$(git rev-parse --show-toplevel)}
 WEIGTH_OUT_PATH=${WEIGTH_OUT_PATH:-""}
 ZKV_NODE_EXE=${ZKV_NODE_EXE:-"${PROJECT_ROOT}/target/production/zkv-node"}
+ZKV_RUNTIME=${ZKV_RUNTIME:-"${PROJECT_ROOT}/target/production/wbuild/zkv-runtime/zkv_runtime.wasm"}
```

Comment:
I don't expect any concrete difference, but maybe we can be consistent and always use the compressed runtime blob?